packages/stream_chat/lib/src/models/filter.dart

```diff
@@ -0,0 +1,197 @@
+// ignore_for_file: non_constant_identifier_names, constant_identifier_names
```

Comment:
👀

Reply:
@Nash0x7E2 It's to suppress lint because of the underscore after `in`, I added the `_` because we do not have any support for [escaping reserved keyword](https://github.com/dart-lang/language/issues/271) in dart.
```dart
  /// Matches any of the values specified in an array.
  in_,
```